pkgs/applications/virtualization/virtualbox/default.nix

```
@@ -229,6 +229,12 @@ in stdenv.mkDerivation {
 
   meta = {
     description = "PC emulator";
+    longDescription = ''
+      VirtualBox is a powerful x86 and AMD64/Intel64 virtualization product for enterprise as well as home use.
```

```suggestion
      VirtualBox is an x86 and AMD64/Intel64 virtualization product for enterprise and home use.
```

I'm sure this is from their content, but let's make it less subjective. Powerful is a pretty bankrupt term.